pkg/acsengine/const.go

```diff
@@ -158,6 +158,11 @@ var KubeConfigs = map[string]map[string]string{
 }
 
 const (
+	// Root URL for extensions
+	DefaultExtensionsRootURL = "https://raw.githubusercontent.com/Azure/acs-engine/master/"
```

Comment:
this file has a bunch of them but these are all urls so sure I can move this one.